AWSSDKSwiftCLI/Package.swift

```diff
@@ -5,28 +5,61 @@ import PackageDescription
 let package = Package(
     name: "AWSSDKSwiftCLI",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v13)
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser", from: "1.2.0"),
         .package(url: "https://github.com/apple/swift-package-manager", from: "0.6.0"),
         .package(url: "https://github.com/apple/swift-algorithms", from: "1.0.0"),
         .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
+        .package(url: "https://github.com/awslabs/aws-sdk-swift", from: "0.46.0"),
```

Comment:
Our own AWS SDK for Swift is added as a dependency so that it can be used to perform the S3 & Cloudfront operations needed for publishing to the Registry.